web/src/components/OpeningHours.tsx

```diff
@@ -56,27 +64,34 @@ const OpeningHours = ({
   appointmentOverlayLink,
 }: OpeningHoursProps): ReactElement | null => {
   const { t } = useTranslation('pois')
+  const theme = useTheme()
+  const isOnlyWithAppointment = !openingHours && !!appointmentOverlayLink
 
   const openingHoursTitle = (
     <TitleContainer>
       <span>{t('openingHours')}</span>
       <OpeningContainer>
         <OpeningLabel $isOpen={isCurrentlyOpen}>
-          {t(getOpeningLabel(isTemporarilyClosed, isCurrentlyOpen))}
+          {t(getOpeningLabel(isTemporarilyClosed, isCurrentlyOpen, openingHours))}
         </OpeningLabel>
       </OpeningContainer>
     </TitleContainer>
   )
-  if (isTemporarilyClosed) {
-    return <TitleContainer>{openingHoursTitle}</TitleContainer>
+  if (isTemporarilyClosed || isOnlyWithAppointment) {
+    return (
+      <>
+        <Spacer $borderColor={theme.colors.borderColor} />
```

Comment:
There is two spacers here:
<img src="https://github.com/user-attachments/assets/599ce4cd-9a54-4357-a468-ff2e1a410ba1" width="300px" />